docs/dev-guide/guide.md

```diff
@@ -174,6 +174,10 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 ## Database stored procedures (sprocs)
 
+!!! warning
+
+    We are migrating away from the use of [sprocs](https://www.geeksforgeeks.org/postgresql/postgresql-introduction-to-stored-procedures/). Prefer writing logic in TypeScript instead.
```

Comment:
I have somewhat of a reflexive aversion to geeksforgeeks. Can we link to something else, either the Postgres docs directly or something like https://neon.com/postgresql/postgresql-plpgsql/postgresql-create-procedure if you want a higher-level description?

Reply:
Done in https://neon.com/postgresql/postgresql-plpgsql/postgresql-create-procedure